contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/pycodestyle_subsystem.py

```diff
@@ -8,10 +8,10 @@
 from pants.contrib.python.checks.tasks.checkstyle.plugin_subsystem_base import PluginSubsystemBase
 
 
-class PEP8Subsystem(PluginSubsystemBase):
-  options_scope = 'pycheck-pep8'
+class PyCodeStyleSubsystem(PluginSubsystemBase):
+  options_scope = 'pycheck-pycodestyle'
```

Comment:
this options scope change is going to break consumers (their options flags and pants.ini would have to change as a result) and would be disallowed under the current deprecation policy here: https://www.pantsbuild.org/deprecation_policy.html#disallowed-api-changes

I think you'll have to deprecate the old scope and alias it to the new one to avoid that (or can just leave this existing scope name untouched).

Reply:
We're currently in `1.8.0.dev0`, so it would be good to target `1.10.0.dev0` with this. Thanks!

Reply:
Updated to 1.10.0.dev0